.env

```diff
@@ -1,3 +1,6 @@
 COVERALLS_REPO_TOKEN=X7TQgZquQIFGx5MnGlv40YL3iS8v6LM66
+S3_BUCKET=soti-user
+S3_KEY=AKIAJIUHYVKULQSNJN4Q
+S3_SECRET=y4W3DiOxMY1uZ8swS5a0DiNpgQV665vVQuqFZFej
```

Comment:
you want to add your .env file to your .gitignore and NOT commit these keys to your repository. If you leak these publicly online someone can use these to charge the credit card they are linked to